backend/api/Services/MissionLoaders/EchoMissionResponse.cs

```diff
@@ -81,7 +81,7 @@ public class SensorType
     public class EchoPose
     {
         [JsonPropertyName("poseId")]
-        public int PoseId { get; set; }
+        public int? PoseId { get; set; }
```

Comment:
Why is this nullable?

Reply:
I saw it was null one time from Echo, and we have a check:
`if (planItem.PoseId is null)` after so that it throws a meaningfull error message instead of crashing when trying to load the json from Echo